stroeer/core/v1/core_article_service.proto

```diff
@@ -115,6 +117,7 @@ message ListArticlesRequest {
   Filters filters = 2;
   int32 page_size = 3;
   string page_token = 4;
+  Legal legal = 5;
```

Comment:
Thought of including the term `filter` either in the enum name or the variable, but the distinction to `Filters` should be clear. Maybe add the new filter options (elements, unpublished articles) to the existing filters?

Reply:
I also like the idea to add the legal stuff to `Filters`